pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -133,6 +137,26 @@ func EnforceHostNetwork() bool {
 	return cephv1.EnforceHostNetwork()
 }
 
+func SetRevisionHistoryLimit(data map[string]string) {
+	strval := k8sutil.GetValue(data, revisionHistoryLimitSettingName, revisionLimitDefaultValue)
+	var numval int64 = 0
+	if strval != "" {
+		var err error
+		numval, err = strconv.ParseInt(strval, 10, 32)
+		if err != nil {
+			logger.Warningf("failed to parse value %q for %q. assuming default value 3.", strval, revisionHistoryLimitSettingName)
+			return
+
+		}
+	}
+	revisionHistoryLimit = int32(numval)
+
+}
+
+func RevisionHistoryLimit() *int32 {
+	return &revisionHistoryLimit
```

Comment:
If the variable is of type `*int32`, this is simplified here
```suggestion
	return revisionHistoryLimit
```